README.md

```diff
@@ -1 +1,28 @@
-# aws-nuke
\ No newline at end of file
+# aws-nuke
+
+This repo contains a CLI tool to delete all AWS resources in an account. The currently supported functionality includes:
```

Comment:
It's probably a good idea to add two more things to this intro section:

1. Motivation: explain why aws-nuke exists and you might want to use it. E.g., you have an account you use for testing and need to clean up left over resources so AWS doesn't charge you for them.

1. Warning: big, bold text explaining that this tool is destructive, irreversibly deletes things in your account, and that you must be very careful with its use.